test/development/acceptance/ReactRefreshLogBox.test.ts

```diff
@@ -814,26 +814,39 @@ describe.each(['default', 'turbo'])('ReactRefreshLogBox %s', () => {
     expect(texts).not.toContain('<unknown>\n<anonymous>')
     expect(texts).toContain('foo\nbar (1:1)')
 
-    // Test that node:internal errors should be hidden
+    await cleanup()
+  })
 
-    next.patchFile(
-      'pages/index.js',
-      // Node.js will throw an error about the invalid URL since it happens server-side
-      outdent`
+  test('should hide unrelated frames in stack trace with node:internal calls', async () => {
+    const { session, browser, cleanup } = await sandbox(
+      next,
+      new Map([
+        [
+          'pages/index.js',
+          // Node.js will throw an error about the invalid URL since it happens server-side
+          outdent`
       export default function Page() {}
       
       export function getServerSideProps() {
         new URL("/", "invalid");
         return { props: {} };
-      }`
+      }`,
+        ],
+      ])
     )
-
+    // Test that node:internal errors should be hidden
```

Comment:
```suggestion
    // Test that node:internal errors should be hidden
```

unnecessary comment since this is a separate test now